README.md

```diff
@@ -11,6 +11,58 @@
 [2]: https://www.stackage.org/package/mailchimp/badge/lts
 [3]: https://www.stackage.org/package/mailchimp/badge/nightly
 
+## Example
+
+To get started, create a manager using a library or the provided
+function to do so.
+
+Right now, this supports basic access authentication, so you need an
+API key.
+
+You need a MailChimp API key. Go to MailChimp and then to your
+account. This is the minimum needed to get started.
+
+In order to see an interesting example, you also need a list ID.
+
+```
+  let
+    MainClient {..} = makeMainClient
+```
+
+A main client includes the required function to create an auth client
+which takes your key.
+
+```
+  let
+    AuthClient {..} = makeAuthClient (BasicAuthData "" key)
```

Comment:
@RobertFischer Sounds good, thanks! I don't remember much of this pull request, so I'll take a look once I get a change and try to merge this quickly.